scripts/README.md

```diff
@@ -31,5 +31,23 @@ This folder is for storing any scripts that may be helpful for using stellar-cor
 - Description - A Python script that compares two CSV files produced by `tracy-csvexport` (which in turn reads output from `tracy-capture`). The purpose of this script is to detect significant performance impacts of changes to stellar-core by capturing before-and-after traces.
 - Usage - Ex. `tracy-capture -o old.tracy -s 10 -a 127.0.0.1` to capture a 10 second trace of stellar-core running on the local machine. Then run `tracy-csvexport -u old.tracy >old.csv`. Then make a change to stellar-core and repeat the process to capture `new.tracy` and `new.csv`. Finally, run `DiffTracyCSV.py --old old.csv --new new.csv` and inspect the differences.
 
+### Soroban Settings Upgrade
+- Directory - `soroban-settings`
+- Name - `SorobanSettingsUpgrade.py`
+- Description - A python script that can setup a setting upgrade or retrieve
+  current settings for Futurenet through Soroban RPC. The next step is to submit all transactions directly to stellar-core's `tx` endpoint. Note that the actual upgrade command will have to be
+  submitted manually on the core nodes. 
+- Prerequisites - run `make build` under `soroban-settings/write_upgrade_bytes`
```

Comment:
`make build` wanted me to install the target `wasm32-unknown-unknown` so we can add a pre-requisite to run `rustup target add wasm32-unknown-unknown`.

Reply:
Updated